apis/nucleus/src/components/listbox/interactions/listbox-keyboard-navigation.js

```diff
@@ -66,6 +74,18 @@ export function getFieldKeyboardNavigation({ select, confirm, cancel, setScrollP
         }
         setScrollPosition?.('end');
         break;
+      case KEYS.A:
+        if (ctrlKey || metaKey) {
```

Comment:
Mac question: should this check metaKey?
It does in the old listbox in sense-client, but the crtl-End + ctrl-Home above don't